workflows/energyplus.py

```diff
@@ -92,93 +92,80 @@ def get_end_summary_from_err(err_file_path):
-        suffixes.append(".mtr")
-        suffixes.append("Proc.csv")
-
-        suffixes.append("_slab.out")
-        suffixes.append(".slab")
-        suffixes.append("_slab.ger")
-
-        suffixes.append("_bsmt.csv")
-        suffixes.append(".edd")
-        suffixes.append("Table.xml")
-        suffixes.append("_perflog.csv")
-
-        # the following were not included in EP-Launch 2
-        suffixes.append(".end")
-        suffixes.append(".sci")
-        suffixes.append(".rvaudit")
-        suffixes.append(".sql")
-        suffixes.append(".log")
-
-        # the rest of these are alternative extensions for the same
-        suffixes.append("Table.csv")
-        suffixes.append("Table.tab")
-        suffixes.append("Table.txt")
-
-        suffixes.append(".tab")
-        suffixes.append(".txt")
-
-        suffixes.append("Meter.tab")
-        suffixes.append("Meter.txt")
-
-        suffixes.append("Zsz.tab")
-        suffixes.append("Zsz.txt")
-        suffixes.append("Ssz.tab")
-        suffixes.append("Ssz.txt")
-
-        suffixes.append("Map.tab")
-        suffixes.append("Map.txt")
-        return suffixes
+        return [
+            # the following are in the same order as the buttons in EP-Launch 2
+            "Table.htm",
+            ".csv",
+            "Meter.csv",
+            ".err",
+            ".rdd",
+            ".mdd",
+            ".eio",
+            ".svg",
+            ".dxf",
+            ".mtd",
+            "Zsz.csv",
+            "Ssz.csv",
+            "DElight.in",
+            "DElight.out",
+            "Map.csv",
+            "DElight.eldmp",
+            "DElight.dfdmp",
+            "Screen.csv",
+            ".expidf",
+            ".epmidf",
+            ".epmdet",
+            ".shd",
+            ".wrl",
+            ".audit",
+            ".bnd",
+            ".dbg",
+            ".sln",
+            "_bsmt.out",
+            ".bsmt",
+            "_bsmt.audit",
+            ".eso",
+            ".mtr",
+            "Proc.csv",
+            "_slab.out",
+            ".slab",
+            "_slab.ger",
+            "_bsmt.csv",
+            ".edd",
+            "Table.xml",
+            "_perflog.csv",
+            ".end",
+            ".sci",
+            ".rvaudit",
+            ".sql",
+            ".log",
+            "Table.csv",
+            "Table.tab",
+            "Table.txt",
+            ".tab",
+            ".txt",
+            "Meter.tab",
+            "Meter.txt",
+            "Zsz.tab",
+            "Zsz.txt",
+            "Ssz.tab",
+            "Ssz.txt",
+            "Map.tab",
+            "Map.txt",
+        ]
 
     def run_energyplus(self, isIP, run_directory, file_name, args, by_api):
 
         full_file_path = os.path.join(run_directory, file_name)
         file_name_no_ext, extension = os.path.splitext(file_name)
-        # the following is the same when .idf is used
+        output_directory = os.path.join(run_directory, f"EPLaunchRun_{file_name_no_ext}")
```

Comment:
We now set up a subdirectory to run EnergyPlus and the auxiliary pre- and post-processors.  This will be named `<original_idf_dir>/EPLaunchRun_<original_idf_name_no_extension>/`.  This is a key piece of allowing EP Launch to run multiple files at once.  EnergyPlus has a tendency of dropping files into the current working directory, and if you allow it to do that while multiple are running then chaos ensues.